docs/upload-strategies/tus.mdx

```diff
@@ -0,0 +1,273 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import UppyCdnExample from '/src/components/UppyCdnExample';
+
+# Tus
+
+The `@uppy/tus` plugin brings resumable file uploading with [Tus](http://tus.io) to Uppy by wrapping the [tus-js-client][].
+
+## When should I use it?
+
+:::tip
+Not sure which upload strategy is best for you? Read “[Choosing the upload strategy you need](/docs/guides/choosing-upload-strategy)”.
+:::
+
+[Tus][tus] is an open protocol for resumable uploads built on HTTP.
+This means accidentally closing your tab or losing connection let’s you continue, for instance, your 10GB upload
+instead of starting all over.
+
+It supports any language, any platform, and any network. You can checkout the [implementations](https://tus.io/implementations.html) to
+find the implementation in your favorite language. You can store files on your self-hosted Tus server, but also use service integrations (such as S3) to store files
+externally.
```

Comment:
```suggestion
Tus consists of a client and a server. Multiple client and server [implementations](https://tus.io/implementations.html) exist in many languages. Uppy uses `tus-js-client` internally and you can set up any server you like to use with Uppy. You can store files on your self-hosted Tus server itself, but more common is to use service integrations (such as S3) to store files externally.

:::tip
If you don't want to host your own Tus server, you can take advantage of Transloadit's hosted services - read “[Choosing the upload strategy you need](/docs/guides/choosing-upload-strategy)”.
:::
```

Reply:
I'm not sure we need to reiterate this here. At the end of the day, this is still an OSS product and the website as well. The guide is already linked 10 lines up and the fact that TL has hosted servers is also mentioned in FAQ. What do you think??

Reply:
I would be more open to this change if we don't use `:::tip` and we don't link to the same thing as the previous tip does. Perhaps this can be one extra line, with a link to the FAQ below?

Reply:
good idea! i updated the comment

Reply:
I think I still prefer the previous text and would only add this new sentence:

```suggestion
It supports any language, any platform, and any network. You can checkout the [implementations](https://tus.io/implementations.html) to
find the implementation in your favorite language. You can store files on your self-hosted Tus server, but also use service integrations (such as S3) to store files
externally. If you don't want to host your own Tus server, you can take advantage of [Transloadit's hosted services](#are-there-hosted-tus-servers). 
```

Reply:
I thought it would be worth mentioning that Tus is a server and a client, because newcomers might be confused as to why they would choose tus in any language when clearly Uppy is a Javascript based library. Whether Tus client supports many different languages/platforms is not so interesting for someone who just wants to get Uppy up and running.

Reply:
> newcomers might be confused as to why they would choose tus in any language when clearly Uppy is a Javascript based library

People first need to setup a Tus server before they can do anything. Just because Uppy is JS, doesn't mean that their backend is, they may have an existing .NET backend. I think it's important to reference the implementations to choose from.

Reply:
yes, that's why i want to mention that tus is a client and a server, so that they understand that they must choose a tus server, not a client

Reply:
I agree with @mifi here. The docs should mention that you need a tus server to use the protocol and plugin for uploading. For of the time, this is clear. But from time to time users still get confused.

Reply:
I incorporated your suggestions but made it a bit more succinct. This is the "Why should you use it section" and some details (like tus-js-client) are already provided in the section above.

```suggestion
Tus supports any language, any platform, and any network. It requires a client and server integration to work. You can checkout the client and server [implementations](https://tus.io/implementations.html) to
find the server in your preferred language. You can store files on the Tus server itself, but you can also use service integrations (such as S3) to store files externally. If you don't want to host your own server, see “[Are there hosted Tus servers?](#are-there-hosted-tus-servers)”.
```